this is with: 

```
arm-none-eabi-gcc (GNU Arm Embedded Toolchain 10.3-2021.07) 10.3.1 20210621 (release)
```

Without the above patch, the following error is generated:

```
/var/folders/3z/1y09kb294wj5jcyqh09x0th00000gn/T//ccQq0fkA.s: Assembler messages:
/var/folders/3z/1y09kb294wj5jcyqh09x0th00000gn/T//ccQq0fkA.s:15: Warning: ignoring changed section attributes for .data
/var/folders/3z/1y09kb294wj5jcyqh09x0th00000gn/T//ccQq0fkA.s:28: Warning: dwarf line number information for .data ignored
/var/folders/3z/1y09kb294wj5jcyqh09x0th00000gn/T//ccQq0fkA.s:33: Warning: dwarf line number information for .data ignored
/var/folders/3z/1y09kb294wj5jcyqh09x0th00000gn/T//ccQq0fkA.s:44: Warning: dwarf line number information for .data ignored
/var/folders/3z/1y09kb294wj5jcyqh09x0th00000gn/T//ccQq0fkA.s:1127: Error: leb128 operand is an undefined symbol: .LVU2
make[1]: *** [built/swd-mux-test-74-0.1/bl/jacdac-stm32x0/bl/blutils.o] Error 1
```

--- a/src/blhw.h
+++ b/src/blhw.h
@@ -49,7 +49,7 @@ struct bl_info_block {
 extern struct bl_info_block bl_info;
 #define bl_dev_info bl_info.devinfo
 
-STATIC_ASSERT(sizeof(struct device_info_block) == 16);
+STATIC_ASSERT_EXT(sizeof(struct device_info_block) == 16, BL_HW);
 
 struct app_top_handlers {
     uint32_t stack_bottom;

